src/postInstall.js

```diff
@@ -0,0 +1,4 @@
+try {
+  const ph = require('customerio-expo-plugin/postinstall');
+  ph.runPostInstall();
+} catch (error) {}
```

Comment:
chef's kiss. This is great! 

I suggest a comment added to the code. Explaining why this postinstall script exists (`npm install` order is undetermined) for future reference. It's not clear in this code why we do this.

Reply:
Sorry, I meant to the code comment should suggest why the RN SDK is trying to run a post install script on the expo plugin. Something such as, "Run the expo plugin post install script because the expo plugin might need to modify the code in this module. We run the expo plugin here and also in the expo plugin since the order of modules in `npm install` is unknown"

The current comment gets outdated if the expo plugin changes what is in the post-install script.